src/test/java/net/helpscout/api/ApiClientTest.java

```diff
@@ -64,6 +77,33 @@ public void shouldReturnDetailedErrorMessageForBadRequest() {
         }
     }
     
+	@Test
+	public void shouldRetrieveSpecificCustomer() throws ApiException {
+		givenThat(get(urlEqualTo("/v1/customers/60984612.json"))
+				.willReturn(aResponse().withStatus(HTTP_OK)
+				.withBody(getResponse("customer"))));
+
+		Long customerId = 60984612L;
+		Customer customer = client.getCustomer(customerId);
+		assertEquals("Peter", customer.getFirstName());
+		assertEquals(customerId, customer.getId());
+		assertNotNull(customer.getAddress().getCreatedAt());
+		Long addressId = 1187643L;
+		assertEquals(addressId, customer.getAddress().getId());
+
+	}
+
+	@SneakyThrows
+	public String getResponse(String responseFile) {
```

Comment:
See comment above about eliminating this method, but if that's not possible then this should probably be private.